src/_posts/platform/app/scaling/2000-01-01-scaling.md

```diff
@@ -0,0 +1,225 @@
+   container:
+   ```bash
+   scalingo --app my-app scale web:1:XL
+   ```
+   The output should look like this:
+   ```
+   Your application is being scaled to:
+     web: 1 - XL
+   ```
+
+### Using the Terraform Provider
+
+1. Set the `size` value of the `scalingo_container_type` `resource` block in
+   your Terraform file to define the size of the container(s) running the
+   corresponding [process type]({% post_url platform/app/2000-01-01-procfile %}):
+   ```tf
+   resource "scalingo_container_type" "web" {
+     app    = scalingo_app.my_app.name
+     name   = "web"
+     amount = 1
+     size   = "XL"
+   }
+   ```
+   In this example, we ask the platform to use `XL` container(s) to run the
+   `web` process type.
+
+
+## Scaling Horizontally
+
+### Using the Dashboard
+
+1. From your web browser, open your [dashboard](https://dashboard.scalingo.com/apps)
+2. Click on the application you want to scale out or in
+3. Click on the **Resources** tab
+4. Locate the **Containers** block
+5. Next to the process type you want to scale up/down, locate the **Qty**
+   dropdown
+6. From the dropdown, select the number of container(s) to start
+7. Click the **Scale** button on the right side
+
+### Using the Command Line
+
+1. Make sure you have correctly [setup the Scalingo command line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, run the following command to view the current
+   formation:
+   ```bash
+   scalingo --app my-app scale
+   ```
+   The output should look like this:
+   ```
+   +--------+--------+------+---------+
+   |  NAME  | AMOUNT | SIZE | COMMAND |
+   +--------+--------+------+---------+
+   | web    |      1 | M    | -       |
+   +--------+--------+------+---------+
+   ```
+   In the example above, the application runs with 1 `web` container of size
+   `M`.
+3. From the command line, run the following command to scale out to 5 `web`
+   containers of size `M`:
+   ```bash
+   scalingo --app my-app scale web:5:M
+   ```
+   The output should look like this:
+   ```
+   Your application is being scaled to:
+     web: 5 - M
+   ```
+
+### Using the Terraform Provider
+
+1. Set the `amount` value of the `scalingo_container_type` `resource` block in
+   your Terraform file to define the size of the container(s) running the
+   corresponding [process type]({% post_url platform/app/2000-01-01-procfile %}):
+   ```tf
+   resource "scalingo_container_type" "web" {
+     app    = scalingo_app.my_app.name
+     name   = "web"
+     size   = "M"
+     amount = 5
+   }
+   ```
+   In this example, we ask the platform to start 5 containers running the `web`
+   process type.
+
+{% note %}
+It is also possible to scale both horizontally **and** vertically at the same
+time using the dashboard, the Command Line or the Terraform Provider.
+{% endnote %}
+
+
+## Monitoring Scaling Events
+
+The following event is available to monitor the Autoscaler executions:
```

Comment:
question/issue: is it really only for the autoscaler? I think it's also for scaling operation done by the user.

Reply:
You're right, it's a bad copy-paste review from me. It's been fixed.